python/src/usecases/oscillator.py

```diff
@@ -0,0 +1,217 @@
+        formula += "(4*zetas*sqrt(ks/ms)^3*(zetap*zetas*(4*(zetap/2+zetas/2)^2+"
+        formula += "((sqrt(kp/mp)-sqrt(ks/ms))/(sqrt(kp/mp)/2+sqrt(ks/ms)/2))^2)+"
+        formula += "(ms/mp)*(zetap/2+zetas/2)^2)*4*(zetap/2+zetas/2)*(sqrt(kp/mp)/2+sqrt(ks/ms)/2)^4))"
+
+        self.model = ot.SymbolicFunction(
+            ["mp", "ms", "kp", "ks", "zetap", "zetas", "Fs", "S0"], [formula]
+        )
+
+        # Mass of primary system
+        self.distributionMp = ot.LogNormalMuSigmaOverMu(
+            self.muMp, self.sigmaOverMuMp
+        ).getDistribution()
+        self.distributionMp.setName("Mass of primary system")
+        self.distributionMp.setDescription(["mp"])
+
+        # Mass of secondary system
+        self.distributionMs = ot.LogNormalMuSigmaOverMu(
+            self.muMs, self.sigmaOverMuMs
+        ).getDistribution()
+        self.distributionMs.setName("Mass of secondary system")
+        self.distributionMs.setDescription(["ms"])
+
+        # Spring stiffness of primary system
+        self.distributionKp = ot.LogNormalMuSigmaOverMu(
+            self.muKp, self.sigmaOverMuKp
+        ).getDistribution()
+        self.distributionKp.setName("Spring stiffness of primary system")
+        self.distributionKp.setDescription(["kp"])
+
+        # Spring stiffness of secondary system
+        self.distributionKs = ot.LogNormalMuSigmaOverMu(
+            self.muKs, self.sigmaOverMuKs
+        ).getDistribution()
+        self.distributionKs.setName("Spring stiffness of secondary system")
+        self.distributionKs.setDescription(["ks"])
+
+        # Damping ratio of primary system
+        self.distributionZetap = ot.LogNormalMuSigmaOverMu(
+            self.muZetap, self.sigmaOverMuZetap
+        ).getDistribution()
+        self.distributionZetap.setName("Damping ratio of primary system")
+        self.distributionZetap.setDescription(["zetap"])
+
+        # Damping ratio of secondary system
+        self.distributionZetas = ot.LogNormalMuSigmaOverMu(
+            self.muZetas, self.sigmaOverMuZetas
+        ).getDistribution()
+        self.distributionZetas.setName("Damping ratio of secondary system")
+        self.distributionZetas.setDescription(["zetas"])
+
+        # Force capacity of secondary spring
+        self.distributionFs = ot.LogNormalMuSigmaOverMu(
+            self.muFs, self.sigmaOverMuFs
+        ).getDistribution()
+        self.distributionFs.setName("Force capacity of secondary spring")
+        self.distributionFs.setDescription(["Fs"])
+
+        # Intensity of white noise excitation
+        self.distributionS0 = ot.LogNormalMuSigmaOverMu(
+            self.muS0, self.sigmaOverMuS0
+        ).getDistribution()
+        self.distributionS0.setName("Intensity of white noise excitation")
+        self.distributionS0.setDescription(["S0"])
+
+        # Joint distribution of the input parameters
+        self.distribution = ot.ComposedDistribution(
```

Comment:
I suggest to create a new attribute with the reference probability with the largest possible number of significant digits. This will allow to create benchmark scripts when needed. Also, please write in the comments of this function how this reference probability was estimated. This may allow to improve the reference when possible.

Reply:
Thanks for your remark. Due to the magnitude order of this probability, estimating with accuracy this probability requires 10^10 or 10^11 samples. So I couldn't do that on my laptop and I prefered to indicate the value of the reference paper.

Reply:
I cannot be sure until I try, but my guess is that an importance sampling algorithm with a SORM approximation may perform correctly, isn'it? This is especially true with a symbolic function, since its derivatives are exact in this case. With that combination of algorithms and implementation, a large sample may not be required to acheive a good accuracy. Am I right?

Reply:
This can be improved afterwards anyway.